### ISSUE TITLE
[Bug]Time format conversion error when resend sms code

### ISSUE THREAD

**Issue by zhanglei0321** [firebase/FirebaseUI-iOS#329]
the wrong code in the file [FUIPhoneVerificationViewController.m](https://github.com/firebase/FirebaseUI-iOS/blob/master/FirebasePhoneAuthUI/FUIPhoneVerificationViewController.m)
 ```
NSTimeInterval minutes = _resendConfirmationCodeSeconds / 60;
NSTimeInterval seconds = (NSUInteger)_resendConfirmationCodeSeconds % 60;
NSString *formattedTime = [NSString stringWithFormat:@"%01.0f:%02.0f", minutes, seconds];
```
when "_resendConfirmationCodeSeconds" is greater than 30, "minutes" is greater than 0.5f,  the value of "formattedTime" is "1:xx",and isnot "0:xx" .
the right code like this:
 ```
NSInteger minutes = _resendConfirmationCodeSeconds / 60;
NSInteger seconds = (NSUInteger)_resendConfirmationCodeSeconds % 60;
NSString *formattedTime = [NSString stringWithFormat:@"%ld:%02ld", minutes, seconds];
```

**Comment by morganchen12** [comment]
I think `seconds` can still be an `NSTimeInterval` in this case if we want to show sub-second precision.

```objc
NSInteger minutes = _resendConfirmationCodeSeconds / 60; // Integer type for truncation
NSTimeInterval seconds = (NSUInteger)_resendConfirmationCodeSeconds % 60;
NSString *formattedTime = [NSString stringWithFormat:@"%ld:%02.0f", minutes, seconds];
```

If we're not showing sub-second precision, maybe we should round the seconds so on average the time is more accurate.

**Comment by morganchen12** [comment]
Fixed in #332.
